bitbots_blackboard/bitbots_blackboard/capsules/costmap_capsule.py

```diff
@@ -69,11 +68,7 @@ def __init__(self, blackboard: "BodyBlackboard"):
         self.calc_gradients()
 
 
-    ############
-    # Obstacle #
-    ############
-
-    def robot_obstacle_callback(self, msg: RobotArray):
+    def robot_callback(self, msg: RobotArray):
         """
         Callback with new obstacles
```

Comment:
The comment is now outdated :D

Reply:
Does this method create a costmap with other robots on it marked as obstacles?

Reply:
The main issue is that the robots are called obstacles and in the comment as well as the body of the function. This is a relict of the past.